docs/ubuntu1604.md

```diff
@@ -0,0 +1,22 @@
+Ubuntu 16.04 prep
+-----------------
+
+The following commands are one possible way to quickly prepare an instance of
+Ubuntu 16.04 for Uno.
+
+```bash
+#install stuff
+sudo apt install --assume-yes openjdk-8-jdk maven git openssh-server wget
+
+#setup passwordless ssh
+ssh-keygen -t rsa -N '' -f ~/.ssh/id_rsa
```

Comment:
I just tried keygen when I had an existing key and it asked me if I wanted to overwrite

```
$ ssh-keygen -t rsa -N '' -f ~/.ssh/id_rsa
Generating public/private rsa key pair.
/home/user42/.ssh/id_rsa already exists.
Overwrite (y/n)? n
```